modules/profile/manifests/st2server.pp

```diff
@@ -655,12 +611,30 @@
   #
   # This is a pretty tight coupling to the st2 puppet module for right now.
   # TODO Fix when it makes sense and it has a home.
-  file { '/etc/init/st2web.conf':
-    ensure  => file,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0444',
-    source  => 'puppet:///modules/st2/etc/init/st2actionrunner.conf',
+
+  case $osfamily {
```

Comment:
I think this section should also follow the same pattern, leveraging `$_init_type` as the case versus the OS Version. This way, we'll get other OS for 'relative' free.